Documentation/maintaining-debian-packages.md

```diff
@@ -90,6 +90,48 @@ history into Debian patches. If the patches are modified, commit the changes.
 
 To start from stratch, use `gbp pq drop`.
 
+### Typical session with patches
```

Comment:
I think we can now remove previous section `## Working with patches`, as your tutorial is more helpful.

Reply:
removed.